syndesis-extension-add-header/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,20 @@
+<configuration>
```

Comment:
Can you ad the 
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE xml>
header to have a clean xml files and avoid warning when importing in Eclipse please?